eZ/Publish/API/Repository/Tests/LocationServiceTest.php

```diff
@@ -2348,19 +2348,19 @@ public function testMoveSubtree()
         $locationService = $repository->getLocationService();
 
         // Load location to move
-        $locationToMove = $locationService->loadLocation($mediaLocationId);
+        $locationToMove = $locationService->loadLocation($demoDesignLocationId);
```

Comment:
Was these changes there from before? Could we drop them (changes in `testMoveSubtree()`)? (seems unrelated)

Reply:
same goes for many of the other changes below

Reply:
Hi @andrerom 
Additionally, I added checking of isContainer status to be performed in the same place.
So, it will be possible to remove it from https://github.com/ezsystems/ezplatform-admin-ui/blob/v1.5.6/src/bundle/Controller/LocationController.php#L130-L135
Most test changes are related to moving under a non-container location (Demo Design).

Not sure why that validation was placed in the admin-ui bundle initially.
If there was a reason for that, I can remove `isContainer` part here.
But I think it still makes sense to have tests updated, so the content will be moved under node with isContainer=true.

Reply:
The addition of `isContainer ` check is good, on that just make sure to add PHP doc on the interface to describve the case.

My comment was purely about the test changes in `testMoveSubtree()` and the existing `testMoveSubtree*()` methods below, are these changes needed to make them pass? Did they move to non contianer?

Reply:
Yes, existing testMoveSubtree is testing moving under "Demo Design", which is not a container. So, I updated them to test moving under "Media" location.

Reply:
ok